docs/notebooks/bsync_examples/Small-Office-Level-2.md

```diff
@@ -0,0 +1,2179 @@
+
+![ESPM](./img/ESPM.png)
+
+
+```python
+# define the benchmark scenario
+bench_sc = bsync.Scenario(
+    bsync.AllResourceTotals(
+        bsync.AllResourceTotal(
+            bsync.AllResourceTotal.SiteEnergyUse(229419.13),  
+            bsync.SiteEnergyUseIntensity(41.71),
+            bsync.EnergyCost(5840.54),
+            ID="AllResourceTotal-Benchmark"
+        )
+    ),
+    bsync.LinkedPremises(
+        bsync.LinkedPremises.Building(
+            bsync.LinkedBuildingID(IDref=b1['ID'])
+        )
+    ),
+    ID="Scenario-Benchmark"
+)
+bench_st = bsync.Scenario.ScenarioType()
+bench = bsync.Benchmark(
+    bsync.BenchmarkType(
+        bsync.PortfolioManager(
+            bsync.PMBenchmarkDate(date(2022, 6, 1))
+        )
+    ),
+    bsync.BenchmarkTool("Portfolio Manager"),
+    bsync.BenchmarkYear(2019),
+    bsync.BenchmarkValue(82.)
+)
+
+# 
+scenarios += bench_sc
+bench_sc += bench_st
+bench_st += bench
+```
+
+### Target Scenario
+Relevant Standard 211 Sections:
+- 6.1.4
+
+Since we used a PM score in the baseline, we will also use that for our target. Let's say we are shooting for a target score of 70.
+
+
+```python
+# define the target scenario in reference to the benchmark scenario
+target_sc = bsync.Scenario(
+    bsync.AllResourceTotals(
+        bsync.AllResourceTotal(
+            bsync.AllResourceTotal.SiteEnergyUse(190348.6),  
+            bsync.SiteEnergyUseIntensity(34.6),
+            bsync.EnergyCost(4846.04),
+            bsync.EnergyCostIndex(0.881),
+            ID="AllResourceTotal-Target"
+        )
+    ),
+    bsync.LinkedPremises(
+        bsync.LinkedPremises.Building(
+            bsync.LinkedBuildingID(IDref=b1['ID'])
+        )
+    ),
+    ID="Scenario-Target"
+)
+target_st = bsync.Scenario.ScenarioType()
+target = bsync.Target(
+    bsync.ReferenceCase(IDref=bench_sc["ID"]),
+    bsync.AnnualSavingsSiteEnergy(39070.53),
+    bsync.AnnualSavingsCost(995),
+    bsync.ENERGYSTARScore(89.),
+)
+
+# 
+scenarios += target_sc
+target_sc += target_st
+target_st += target
+```
+
+### Current Building Modeled Scenario
+Relevant Standard 211 Sections:
+- 6.1.5 & 6.1.6
+
+Although not explicitly called out in Standard 211, the current building modeled scenario is mostly implied as part of a Level 2 energy audit when doing more detailed savings estimates / calculations for potential measure(s) implementation. This is because when an energy / cost savings claim is made for a package of measures scenario, it needs to be _in reference_ to something, i.e. a current building modeled scenario (also often referred to as a baseline modeled scenario). The baseline modeled scenario should be interpreted as the expected performance of your building on an average or typical year. This is assuming the baseline modeled scenario is performed with TMY3 data, although they are likely first calibrated with AMY data.
```

Comment:
```suggestion
Although not explicitly called out in Standard 211, the current building modeled scenario is implied as part of a Level 2 energy audit when doing detailed savings estimates and calculations for potential energy conservation measures. This is because when an energy and cost savings claim is made for a package of measures scenario, it needs to be _in reference_ to something, i.e., a current building modeled scenario (also often referred to as a baseline modeled scenario). The baseline modeled scenario should be interpreted as the expected performance of your building on an average or typical year. This is assuming the baseline modeled scenario is performed with TMY data, although they are likely first calibrated with AMY data.
```